staticClassIssue

iamronakshah opened the pull request:
updated String Constant as per issue  #70 

File Changed:
StringConstant.cs: updated all static property to const.

chintans commented on Promact.Oauth.Server/src/Promact.Oauth.Server/Constants/StringConstant.cs:
```diff
@@ -7,85 +7,85 @@ namespace Promact.Oauth.Server.Constants
-        public static string UserEmail = "${{Email}}$";
-        public static string UserPassword = "${{Password}}$";
-        public static string DefaultUserPassword = "User@123";
+        public const string ConsumerAppNameDemo = "Demo Name";
```
We are not using static as well as const. It should be DI friendly

iamronakshah replied:
I have injected string constant class in controller , repository and test cases. now it should be DI friendly.

iamronakshah commented:
Added IStringConstant as per issue #70

Files Changed are: 
Added File : IStringConstant.cs file 

Updated Files are
StringConstant.cs : remove const 
BaseProvider.cs : register Istringconstant 
StartUp.cs :  register Istringconstant 

Added IstringConstant in constructor to resolved dependency and all methods are updated which are use directly stringConstant class.

Updated test case files are
ConsumerAppRepositoryTest.cs , OAuthRepositoryTest.cs ,ProjectTests.cs ,UserRepositoryTest.cs 

Updated Controller files are 
AccountController.cs , OAuthController.cs 

Updated Repository files are 
ConsumerAppRepository.cs  ,OAuthRepository.cs  ,ProjectRepository.cs  , UserRepository.cs
